tests/fixtures/angular-17/angular.json

```diff
@@ -41,13 +39,13 @@
               "budgets": [
                 {
                   "type": "initial",
-                  "maximumWarning": "500kB",
-                  "maximumError": "1MB"
+                  "maximumWarning": "500kb",
+                  "maximumError": "1mb"
                 },
                 {
                   "type": "anyComponentStyle",
-                  "maximumWarning": "2kB",
-                  "maximumError": "4kB"
+                  "maximumWarning": "2kb",
+                  "maximumError": "4kb"
```

Comment:
![image](https://github.com/netlify/angular-runtime/assets/3572990/7ac61acd-17f3-4bf6-a8f6-ee236ede9931)
🤔 I cannot see the difference between these two

Reply:
Hah, took me a while staring at that. It's the casing!

Reply:
Sorry for the diff noise in the angular-17 (previously angular-18) fixture directory. The angular cli update tool doesn't work for downgrading so I ended up just copying a previous fixture from the git history and fully upgrading it to 17. It created some noise in the PR diff since the fixture wasn't created exactly the same way and at the same time.